pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.480.3</version>
+        <version>1.580.3</version>

```

Comment:
I'm not 100% sure why you need the core update. But maybe fine since I was about updating the core dependency during parent POM upgrade